tap_mavenlink/schemas/user_custom_field_values.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
This file looks the same as `story_custom_field_values.json`. 
Part of me thinks, 'Why not try to have one file rather than two?', but the other part of me doesn't mind 🤔 

I'm happy to keep it simple and have two files 👍

Reply:
Happy to keep it simple :+1: